napari/layers/points/_points_key_bindings.py

```diff
@@ -56,63 +58,86 @@ def paste(layer: Points):
     layer._paste_data()
 
 
-@register_points_action(
-    trans._("Select all points in the current view slice."),
-)
-def select_all_in_slice(layer: Points):
-    new_selected = set(layer._indices_view[: len(layer._view_data)])
+def select_data(layer: Points, all_slices=True, shown=False):
+    # If all points are already selected, deselect all points
```

Comment:
I think it would be great to have a docstring here with explanations of what this does before the final merge.